frontend/src/components/teacher/results/StudentAnswersSection/question-elements/CorrectedFraction.tsx

```diff
@@ -15,29 +15,51 @@ const CorrectedFraction = ({
   studentAnswer,
   correctAnswer,
 }: CorrectedFractionProps) => {
-  const [studentAnswerNumerator, studentAnswerDenominator] =
-    studentAnswer ?? [];
+  const isMixedFraction = correctAnswer.wholeNumber !== null;
+
+  let studentAnswerWholeNumber,
+    studentAnswerNumerator,
+    studentAnswerDenominator;
+  if (isMixedFraction) {
+    [
+      studentAnswerWholeNumber,
+      studentAnswerNumerator,
+      studentAnswerDenominator,
+    ] = studentAnswer ?? [];
+  } else {
+    [studentAnswerNumerator, studentAnswerDenominator] = studentAnswer ?? [];
+  }
 
   return (
     <CorrectedInputWrapper
       isWrongAnswer={
         !studentAnswer ||
+        (isMixedFraction &&
+          studentAnswerWholeNumber !== correctAnswer.wholeNumber) ||
         studentAnswerNumerator !== correctAnswer.numerator ||
         studentAnswerDenominator !== correctAnswer.denominator
       }
       w={400}
     >
-      <VStack alignItems="center">
-        <CorrectedTextField
-          correctAnswer={correctAnswer.numerator}
-          studentAnswer={studentAnswerNumerator}
-        />
-        <Divider borderBottomWidth="2px" borderColor="grey.300" />
-        <CorrectedTextField
-          correctAnswer={correctAnswer.denominator}
-          studentAnswer={studentAnswerDenominator}
-        />
-      </VStack>
+      <HStack justifyContent="center">
+        {isMixedFraction && (
+          <CorrectedTextField
+            correctAnswer={correctAnswer.wholeNumber as number}
+            studentAnswer={studentAnswerWholeNumber}
+          />
+        )}
+        <VStack alignItems="center">
+          <CorrectedTextField
+            correctAnswer={correctAnswer.numerator}
+            studentAnswer={studentAnswerNumerator}
+          />
+          <Divider borderBottomWidth="2px" borderColor="grey.300" />
+          <CorrectedTextField
+            correctAnswer={correctAnswer.denominator}
+            studentAnswer={studentAnswerDenominator}
+          />
+        </VStack>
+      </HStack>
     </CorrectedInputWrapper>
   );
 };
```

Comment:
Have we added the implementation for this? (i.e if we add fraction type to a test and then a student takes the test, will it show up in the `Post-Assessment Teacher Experience`?

Or only created the component?

Reply:
it's implemented
<img width="439" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/16892931-d733-4f1a-a7b1-b50b594a8771">
<img width="433" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/ded222ee-19cc-470e-a34c-121bd281823b">